rust-symcrypt/src/hash.rs

```diff
@@ -1,59 +1,98 @@
+/* Hashing functions. For further documentation please refer to symcrypt.h */
```

Comment:
```suggestion
//! Hashing functions. For further documentation please refer to symcrypt.h
```

This makes it a module-level doc comment, so it will show up in docs.rs.

Reply:
Noted, I will change for other files as well